Library/UIAlertController.swift

```diff
@@ -84,39 +84,6 @@ public extension UIAlertController {
     return alertController
   }
 
-  public static func rating(yesHandler: @escaping ((UIAlertAction) -> Void),
-                            remindHandler: @escaping ((UIAlertAction) -> Void),
-                            noHandler: @escaping ((UIAlertAction) -> Void)) -> UIAlertController {
-    let alertController = UIAlertController(
-      title: Strings.profile_settings_rating_title() ,
-      message: Strings.profile_settings_rating_message() ,
-      preferredStyle: .alert
-    )
-    alertController.addAction(
-      UIAlertAction(
-        title: Strings.profile_settings_rating_option_rate_now() ,
```

Comment:
Looks like these 3 strings are no longer being used. Wanna get rid of those?

Reply:
Ah thanks, I'll have to open a PR in our strings for that 👍